layers/+misc/multiple-cursors/packages.el

```diff
@@ -34,3 +34,25 @@
           (define-key state-map (car keybinding) (cdr keybinding))))
       (add-hook 'prog-mode-hook 'turn-on-evil-mc-mode)
       (add-hook 'text-mode-hook 'turn-on-evil-mc-mode))))
+
+(defun multiple-cursors/init-multiple-cursors ()
+  (use-package multiple-cursors
+    :defer t
```

Comment:
Should `:defert t` be removed?
The `evil-mc` use-package section doesn't have it.

And the `multiple-cursors` package might need to be loaded for prog and text mode with hooks like `evil-mc`:
```
(add-hook 'prog-mode-hook 'turn-on-evil-mc-mode)
(add-hook 'text-mode-hook 'turn-on-evil-mc-mode)
```

Reply:
`:defer t` should remain I suppose. I don't know why `evil-mc` does not have it. Probably it is initialized with other evil stuff and deferring it would have no effect.

`use-package` documentation says:
`If you aren't using :commands, :bind, :bind*, :bind-keymap, :bind-keymap*, :mode, :interpreter, or :hook (all of which imply :defer; see the docstring for use-package for a brief description of each), you can still defer loading with the :defer keyword ...`

`mc` is auto-loaded when it is first invoked with key binding.

Reply:
Got it, thanks for the info.